<?xml version="1.0"?>

<launch>

    <node name="velocity_planner" pkg="velocity_planner" type="velocity_planner" output="screen">         
        <param name = "delay_in_sec" value ="1" />
        <param name = "dt" value ="0.1"/>
        <param name = "lag_tau" value = "0.6"/>
        <param name = "Q_vel" value = "1.0"/> 
        <param name = "Q_dis" value =" 4.0"/>
        <param name = "R_weight" value = "0.5"/>
        <param name = "d_time" value ="5.0"/>
        <param name = "d_safe" value ="10.0"/>
    </node>
</launch>